### PR TITLE
bugfix #1986 - replace System.out/err with LOG from @Sl4j

### PR DESCRIPTION
**Pull request by mohangpatil**
### Describe your changes :

I worked on the ..... because ...

#
### Type of change :

- [x] Bug fix
- [x] Improvement
- [x] New feature
- [] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [] Documentation

#
### Frontend Preview (Screenshots) :
<p align="center">For frontend related change, please link screenshots of your changes preview! Optional for backend related changes.
</p>

#
### Checklist:

- [x] I have read the [**CONTRIBUTING**](https://docs.open-metadata.org/open-source-community/developer) document.
- [x] I have commented on my code, particularly in hard-to-understand areas.
- [x] I have added tests that prove my fix is effective or that my feature works.
- [x] All new and existing tests passed.

#

### CODE REVIEW

File: catalog-rest-service/src/main/java/org/openmetadata/catalog/util/TablesInitializer.java [hunk 1]
```diff
@@ -289,16 +292,16 @@ private static void usage() {
 
   private static void printToConsoleInDebug(String message) {
     if (DEBUG_MODE) {
-      System.out.println(message);
```
Comment (harshach) [inline catalog-rest-service/src/main/java/org/openmetadata/catalog/util/TablesInitializer.java:292]: @mohangpatil TablesInitializer is a command line util. We need System.out.println here please revert the changes to the file

Comment (mohangpatil) [reply catalog-rest-service/src/main/java/org/openmetadata/catalog/util/TablesInitializer.java:292]: Sure, thank you for quick reply. I saw it was reported in sonar, so changed it. Reverting it now.

**Comment by harshach** [comment]
> **The Java checkstyle failed.**
> 
> Please run `mvn googleformatter:format@reformat-sources` in the root of your repository and commit the changes to this PR. You can also use [pre-commit](https://pre-commit.com/) to automate the Java code formatting.
> 
> You can install the pre-commit hooks with `make install_test precommit_install`.

@mohangpatil can you run the above

**Comment by mohangpatil** [comment]
Will run pre-tests and pre-commit and will open a new one
